plugwise_usb/helpers/cache.py

```
@@ -72,13 +74,17 @@ def _get_writable_os_dir(self) -> str:
         if os_name == "nt":
             if (data_dir := os_getenv("APPDATA")) is not None:
                 return os_path_join(data_dir, CACHE_DIR)
-            raise CacheError("Unable to detect writable cache folder based on 'APPDATA' environment variable.")
+            raise CacheError(
+                "Unable to detect writable cache folder based on 'APPDATA' environment variable."
+            )
         return os_path_join(os_path_expand_user("~"), CACHE_DIR)
 
     async def write_cache(self, data: dict[str, str], rewrite: bool = False) -> None:
-        """"Save information to cache file."""
+        """ "Save information to cache file."""
```

Ruff (0.11.9)

No whitespaces allowed surrounding docstring text

(D210)